SPECS/emacs/emacs.spec

```diff
@@ -62,6 +77,38 @@ rm %{buildroot}%{_datadir}/applications/*.desktop
 
 mkdir -p %{buildroot}%{_datadir}/emacs/site-lisp/site-start.d
 
+# Install rpm macro definition file
+mkdir -p %{buildroot}%{_rpmconfigdir}/macros.d
+install -p -m 0644 macros.emacs %{buildroot}%{_rpmconfigdir}/macros.d/
+
+mkdir -p %{buildroot}%{site_lisp}
+install -p -m 0644 %SOURCE1 %{buildroot}%{site_lisp}/site-start.el
+install -p -m 0644 %SOURCE2 %{buildroot}%{site_lisp}
+
+# This solves bz#474958, "update-directory-autoloads" now finally
+# works the path is different each version, so we'll generate it here
+echo "(setq source-directory \"%{_datadir}/emacs/%{version}/\")" \
+ >> %{buildroot}%{site_lisp}/site-start.el
+
+mkdir -p %{buildroot}%{site_lisp}/site-start.d
```

Comment:
I think this is already done by `mkdir -p %{buildroot}%{_datadir}/emacs/site-lisp/site-start.d` a few lines above. I'd remove one of them.

Reply:
Removed it